rusk-wallet/src/bin/command.rs

```diff
@@ -103,6 +106,119 @@ pub(crate) enum Command {
         gas_price: Lux,
     },
 
+    /// Start staking DUSK through phoenix
+    PhoenixStake {
+        /// Phoenix Address from which to stake DUSK [default: first address]
+        #[clap(short = 's', long)]
+        addr: Option<Address>,
+
+        /// Amount of DUSK to stake
+        #[clap(short, long)]
+        amt: Dusk,
+
+        /// Max amt of gas for this transaction
+        #[clap(short = 'l', long, default_value_t= DEFAULT_STAKE_GAS_LIMIT)]
+        gas_limit: u64,
+
+        /// Price you're going to pay for each gas unit (in LUX)
+        #[clap(short = 'p', long, default_value_t= DEFAULT_PRICE)]
+        gas_price: Lux,
+    },
+
+    /// Phoeinx Unstake a key's stake
+    PhoenixUnstake {
+        /// Phoenix Address from which your DUSK was staked [default: first
+        /// address]
+        #[clap(short, long)]
+        addr: Option<Address>,
+
+        /// Max amt of gas for this transaction
+        #[clap(short = 'l', long, default_value_t= DEFAULT_STAKE_GAS_LIMIT)]
+        gas_limit: u64,
+
+        /// Price you're going to pay for each gas unit (in LUX)
+        #[clap(short = 'p', long, default_value_t= DEFAULT_PRICE)]
+        gas_price: Lux,
+    },
+
+    /// Phoenix Withdraw accumulated reward for a stake key
+    PhoenixWithdraw {
+        /// Phoenix Address from which your DUSK was staked [default: first
+        /// address]
+        #[clap(short, long)]
+        addr: Option<Address>,
+
+        /// Max amt of gas for this transaction
+        #[clap(short = 'l', long, default_value_t= DEFAULT_STAKE_GAS_LIMIT)]
+        gas_limit: u64,
+
+        /// Price you're going to pay for each gas unit (in LUX)
+        #[clap(short = 'p', long, default_value_t= DEFAULT_PRICE)]
+        gas_price: Lux,
+    },
+
+    /// Deploy a contract using phoenix transaction
+    PhoenixContractDeploy {
+        /// Phoenix Address from which to deploy the contract [default: first]
+        #[clap(short, long)]
+        addr: Option<Address>,
+
+        /// Path to the wasm contract code
+        #[clap(short, long)]
+        code: PathBuf,
+
+        /// Arguments for init function
+        #[clap(short, long)]
+        init_args: Vec<u8>,
+
+        /// Max amt of gas for this transaction
+        #[clap(short = 'l', long, default_value_t= DEFAULT_STAKE_GAS_LIMIT)]
+        gas_limit: u64,
+
+        /// Price you're going to pay for each gas unit (in LUX)
+        #[clap(short = 'p', long, default_value_t= DEFAULT_PRICE)]
+        gas_price: Lux,
+    },
+
+    /// Call a contract using phoenix
+    PhoenixContractCall {
+        /// Phoenix Address from which to call the contract [default: first]
+        #[clap(short, long)]
+        addr: Option<Address>,
+
+        /// ContractId to call
+        #[clap(short, long)]
+        contract_id: Vec<u8>,
+
+        /// Fn name to call
+        #[clap(short, long)]
+        fn_name: String,
+
```

Comment:
`fn_args` are missing documentation, there are the fn arguments to call the contract function with